src/icons/SimpleArrow.tsx

```diff
@@ -0,0 +1,19 @@
+export default function () {
```

Comment:
Let's use `Arrow.tsx` with prop `plain` or `white`, simply change the colors inside the `svg`